node/node.go

```diff
@@ -1079,6 +1080,10 @@ func (n *Node) CreateTransaction(
 		return nil, nil, errors.New("could not fetch signature bytes")
 	}
 
+	if len(value) > n.maxTransactionValueLength {
```

Comment:
What about the other fields? Example: receiver, sender, dataField, signatureHex and chainID?